pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -368,6 +368,8 @@ type KubeletConfiguration struct {
 	// runtimeRequestTimeout is the timeout for all runtime requests except long running
 	// requests - pull, logs, exec and attach.
 	RuntimeRequestTimeout unversioned.Duration `json:"runtimeRequestTimeout"`
+	// If no pulling progress is made before the deadline imagePullProgressDeadline, the image pulling will be cancelled.
```

Comment:
Is there a default value, and can this be turned off?  If so, add those as additional sentences to this godoc.